frontend/src/main/web/lib/components/DropDown.jsx

```diff
@@ -24,6 +23,7 @@ var DropDown = React.createClass({
   render: function() {
     var options = this.props.options,
       selected = this.props.selectedOption,
+      onClickFn = this.props.onClick,
```

Comment:
It doesn't happen every click. It should be called `this.props.onSelectionChange`.